saleor/graphql/webhook/subscription_types.py

```diff
@@ -561,8 +561,8 @@ class GiftCardSent(SubscriptionObjectType, GiftCardBase):
     )
 
     class Meta:
-        root_type = "GiftCard"
-        enable_dry_run = True
+        root_type = None
```

Comment:
This results in removing the `doc_category` in the GraphQL schema. To fix this, we need to add the following to the meta class as well:

```
doc_category = DOC_CATEGORY_GIFT_CARDS
```